tests/xfstests/partition.pm

```diff
@@ -18,18 +18,69 @@ use base 'opensusebasetest';
 use utils;
 use testapi;
 
+sub str_to_mb {
+    my $str = shift;
```

Comment:
Another option is `my ($str) = @_;`

Reply:
I see, I wonder is there any benefit to use @_ than shift :)

Reply:
From my perspective: It avoid using implicit variables (but it's quite common in perl) and  you could use multiple parameters in one command.

But in this specific case, I do not see a benefit.

Reply:
Thanks! I totally understand now.